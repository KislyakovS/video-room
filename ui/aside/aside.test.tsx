import { render } from '@testing-library/react';

import Aside from './aside';

describe('<Aside>', () => {
    it('snapshot', () => {
        const { container } = render(<Aside>Aside</Aside>);

        expect(container).toMatchSnapshot();
    });

    it('should have a sticky class', () => {
        const { getByRole } = render(<Aside sticky>Aside</Aside>);
        const aside = getByRole('complementary');

        expect(aside.classList.contains('sticky')).toBeTruthy();
    });
})