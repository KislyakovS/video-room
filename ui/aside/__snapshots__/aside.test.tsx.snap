// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aside> snapshot 1`] = `
<div>
  <aside
    class="aside"
  >
    Aside
  </aside>
</div>
`;
